### DISCUSSION TITLE
Rustdesk 1.2.7 on Android via work profile

### DISCUSSION THREAD

**Discussion by yaroz** [rustdesk/rustdesk#8986] [General]
We're using the Pro server, so I built the client on the admin site.  I then uploaded it to our private Google store so I could add it through Intune.  I installed it on my Android 12 phone, and went through all of the same steps that I did to get it to work on a non-work profile phone.  I know there's another issue (5441) that deals with this, but it hasn't been addressed.

One thing to note is that when I try to go into the Accessibility menu, it says "You're using this app outside of your work profile".  Anydesk has the same issue, but if we install a plugin in the "personal" side, it seems to work.

**Comment by basilgello** [comment]
You can try https://stackoverflow.com/questions/63066398/i-need-an-adb-command-to-enable-an-android-accesibility-option for your work profile user (10). It does support `--user` modifier :)
